### ISSUE TITLE
1.3.3 pre-release: missing 1 required positional argument: 'error_code'

### ISSUE THREAD

**Issue by m0wlheld** [leeyuentuen/polestar_api#61]
New with pre-release 1.3.3 (which seems to be revoked):

```
2024-01-03 13:09:04.118 ERROR (MainThread) [custom_components.polestar_api.polestar] Unexpected Error on update data PolestarAuthException.__init__() missing 1 required positional argument: 'error_code'
2024-01-03 13:09:04.118 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up polestar_api platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 361, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/polestar_api/sensor.py", line 397, in async_setup_entry
    sensors = [
              ^
  File "/config/custom_components/polestar_api/sensor.py", line 398, in <listcomp>
    PolestarSensor(device, description) for description in POLESTAR_SENSOR_TYPES
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/polestar_api/sensor.py", line 416, in __init__
    unique_id = device.vin[-4:]
                ~~~~~~~~~~^^^^^
TypeError: 'NoneType' object is not subscriptable
```

The error is not present with 1.3.2

**Comment by m0wlheld** [comment]
Fixed with 1.3.4 pre-release

**Comment by leeyuentuen** [comment]
you are very fast to download the new package, I need to tag them before I can test on my production. otherwise I can only test on development xD
